Simplified/Reader2/Internal/ReaderModule.swift

```diff
@@ -14,120 +15,76 @@ import UIKit
 import R2Shared
 
 
-/// The ReaderModule handles the presentation of publications to be read by the
-/// user. It contains sub-modules implementing ReaderFormatModule to handle each
-/// format of publication (eg. CBZ, EPUB).
+/// The ReaderModule handles the presentation of publications to be read by the user.
+/// It contains sub-modules implementing ReaderFormatModule to handle each format of publication (eg. CBZ, EPUB).
 protocol ReaderModuleAPI {
-
+  
   var delegate: ReaderModuleDelegate? { get }
-
-
+  
   /// Presents the given publication to the user, inside the given navigation controller.
-  ///
-  /// - Parameter publication: The R2 publication to display.
-  /// - Parameter book: Our internal book model related to the `publication`.
-  /// - Parameter navigationController: The navigation stack the book will be
-  /// presented in.
   /// - Parameter completion: Called once the publication is presented, or if an error occured.
-  func presentPublication(_ publication: Publication,
-                          book: NYPLBook,
-                          in navigationController: UINavigationController,
-                          completion: @escaping () -> Void)
-
+  func presentPublication(publication: Publication, book: NYPLBook, in navigationController: UINavigationController, completion: @escaping () -> Void)
+  
 }
 
 protocol ReaderModuleDelegate: ModuleDelegate {
-
```

Comment:
are we keeping ReaderModuleDelegate because we might need it for Adobe DRM?

Reply:
I don't think we need it for Adobe DRM... we're not loading DRMs now, so I think it's safe to remove it